Exploratory_Data_Analysis/Principles_of_Analytic_Graphs/lesson

```diff
@@ -139,7 +139,7 @@
   Output: The fifth principle of graphing involves describing and documenting the evidence with sources and appropriate labels and scales. Credibility is important so the data graphics should tell a complete story. Also, using R, you want to preserve any code you use to generate your data and graphics so that the research can be replicated if necessary. This allows for easy verification or finding bugs in your analysis.
 
 - Class: text
-  Output: The sixth and final principle of analytic graphing is maybe the most important. Content is King! If you don't have something interesting to report, your graphs won't save you. Analytical presentations ultimately stand or fall depending on the quality, relevance, and integrity of their content.
```

Comment:
The lowercase "king" is correct, see [this page](http://www.uwyo.edu/english/eighteenth-century-life/style_guide-4.html) - under **Titles of Royalty**.